### DISCUSSION TITLE
Force Heating

### DISCUSSION THREAD

**Discussion by adi90x** [jmcollin78/versatile_thermostat#211] [Q&A]
Hello,

What is the best way to force heating while keeping vtherm ?
At the moment, I have an automation that switch vtherm havac off switch on my heater and then after xx min switch off heater and reenable vtherm.
Is it the best approach ?
This is use either combine with a humidy sensor or with a towel heater, to make sure it heat after a shower is detected.
Curious to hear how you bypass your climate !

**Comment by jmcollin78** [comment]
Why do you need to heat after a shower? For my curiosity? To remove humidity? Isnt'it better to renew the air (open window, VMC, ...) In that case ?

**Comment by adi90x** [comment]
Sometime for humidy, but in all case my heater in the bathroom is a towel
dryer, so I want to have it run a bit to dry the towel !
For humidity, yes opening or vmc is much better !

Le dim. 19 nov. 2023, 19:50, Jean-Marc Collin ***@***.***> a
écrit :

**Comment by jmcollin78** [comment]
ok i've got it. No I don't have any clean solution. What you can try is certainly to force temperature in manual to let says 25° during a few minutes and then come back to normal preset. This could be done with an Automation triggered by a button. This should be simplier to implements and avoid starting the heating in the back of the VTherm so you can view it on the card.

It is perhaps a good improvements to propose. Can you please open an issue with this idea : Force heating during a programmed duration and come back to normal preset. And please explain your case. I'm sure there is other cases relative to this function.
It's a kind of "Boost 1H".

**Comment by adi90x** [comment]
Hello,
Done with #212

Good evening !
